config/prow/plugins.yaml

```diff
@@ -261,43 +261,15 @@ slack:
     exempt_users:
     - k8s-ci-robot # future home of tide
     - k8s-release-robot # anago
+    - cpanato # Release Manager
+    - feiskyer # Release Manager
+    - hasheddan # subproject owner / Release Manager
+    - idealhack # Release Manager
+    - justaugustus # subproject owner / Release Manager
+    - puerco # Release Manager
+    - saschagrunert # subproject owner / Release Manager
+    - xmudrii # Release Manager
```

Comment:
This would exempt these users from all branches, including master/default. Why would we want this?

Reply:
@cblecker -- Thanks for the review!

That actually got me thinking...
We don't actually require exemption anymore, as Release Managers no longer fast-forward branches. 🙃 

I've dropped all of the Release Manager-specific branch exemptions.
/hold cancel